adal/src/telemetry/java/com/microsoft/aad/adal/CacheEvent.java

```diff
@@ -36,6 +36,12 @@
         setProperty(EventStrings.EVENT_NAME, eventName);
     }
 
+    void setSpeRing(final String speRing) {
+        if (null != speRing) {
```

Comment:
Should we report empty SPE ring?

Reply:
I wasn't planning on it. Is there any reason we should?

Reply:
I'm not reporting it in iOS for payload size optimization, I'm wondering if I should :)